l10n_ch_qr_bill_scan/readme/INSTALL.rst

```diff
@@ -1,11 +1,11 @@
 This module depends on `account_invoice_import` which is located in `edi` and has dependencies on `community-data-files`.
-So the repository `edi` and `community-data-files` must be present on the system.
+So the repositories `edi` and `community-data-files` must be present on the system.
 
 Optionally you can also import Vendor bills directly from a PDF.
 To enable this feature those additional dependencies must be
 installed on your Odoo server:
 
-    pip install pyzbar,numpy,pdf2image,opencv-python
+    pip install pyzbar,numpy,pdf2image
```

Comment:
`opencv-python` is removed from there but it still listed in `external_dependencies` in the manifest file, is it a mistake?

Reply:
Hello @sebalix turns out there is actually a better option: `opencv-python-headless` (official), that is built for servers (and doesn't need `libgl1` deb package):

![image](https://github.com/OCA/l10n-switzerland/assets/511258/2a3f5fe8-add9-4a53-bdc1-85e1100621ee)

@sonhd91 just updated the branch